src/main/java/com/iota/iri/service/dto/GetNodeInfoResponse.java

```diff
@@ -56,70 +61,160 @@ public static AbstractResponse create(String appName, String appVersion, int jre
 	}
 
+    /**
+     * The total amount of memory in the Java virtual machine.
+     *
+     * @return The total memory of the JRE.
+     */
 	public long getJreTotalMemory() {
 		return jreTotalMemory;
 	}
 
+    /**
+     * The JRE version this node runs on
+     *
+     * @return The JRE version.
+     */
 	public String getJreVersion() {
 		return jreVersion;
 	}
 
+    /**
+     * The hash of the latest transaction that was signed off by the coordinator.
+     *
+     * @return The latest milestone.
+     */
 	public String getLatestMilestone() {
 		return latestMilestone;
 	}
 
+    /**
+     * Index of the latest milestone.
+     *
+     * @return The latest milestone index.
+     */
 	public int getLatestMilestoneIndex() {
 		return latestMilestoneIndex;
 	}
 
+    /**
+     * The hash of the latest transaction which is solid and is used for sending transactions. 
+     * For a milestone to become solid your local node must basically approve the subtangle of coordinator-approved transactions, 
+     *  and have a consistent view of all referenced transactions.
+     *
+     * @return The latest subtangle milestone index.
+     */
 	public String getLatestSolidSubtangleMilestone() {
 		return latestSolidSubtangleMilestone;
 	}
 
+    /**
+     * Index of the latest solid subtangle.
+     * 
+     * @return The latest subtangle milestone index.
+     */
 	public int getLatestSolidSubtangleMilestoneIndex() {
 		return latestSolidSubtangleMilestoneIndex;
 	}
 
+    /**
+     * Gets the start of the milestone index
+     *
+     * @return The start of the milestone index.
+     */
 	public int getMilestoneStartIndex() {
 		return milestoneStartIndex;
 	}
 
+    /**
+     * Number of neighbors you are directly connected with.
+     *
+     * @return The neighbors.
+     */
 	public int getNeighbors() {
 		return neighbors;
 	}
 
+    /**
+     * Packets which are currently queued up.
+     *
+     * @return The size of the packets queue.
+     */
 	public int getPacketsQueueSize() {
 		return packetsQueueSize;
 	}
 
+    /**
+     * Current UNIX timestamp.
+     *
+     * @return The time.
+     */
 	public long getTime() {
 		return time;
 	}
 
+    /**
+     * Number of tips in the network.
+     *
+     * @return The tips.
+     */
 	public int getTips() {
 		return tips;
 	}
 
+    /**
+     * When a node receives a transaction from one of its neighbors, this transaction is referencing two other transactions t1 and t2 (trunk and branch transaction). 
+     * If either t1 or t2 (or both) is not in the node's local database, then the transaction hash of t1 (or t2 or both) is added to the queue of the "transactions to request".
+     * At some point, the node will process this queue and ask for details about transactions in the "transaction to request" queue from one of its neighbors. 
+     * By this means, nodes solidify their view of the tangle (i.e. filling in the unknown parts).
+     *
+     * @return The transactions to tequest.
```

Comment:
Can you set your IDE to have a margin of 120 columns?

typo tequest
`@return the number of requested transactions`?

Reply:
Set, will update gradually as I edit